.github/workflows/ci_tests_legacy.yaml

```
@@ -12,7 +12,7 @@ on:
   # push:
   #   branches: [ main ]
   # Uncomment the 'pull_request' line below to trigger the workflow in PR
-  # pull_request:
+  pull_request:
```

Reminder to revert this change before merging.

```suggestion
  # pull_request:
```